CHANGELOG.md

```diff
@@ -4,10 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.0.2] - 2020-04-30
```

Comment:
The changelog entry itself is great.

I'm just a bit puzzled by the versions in this file tho (which was there).
RStudio releases here (and in quay.io) follow a different way of versioning to also include version of R/Python.

I wonder if in the case of this repository we never used the CHANGELOG and relied on the [GitHub releases](https://github.com/ministryofjustice/analytics-platform-rstudio/releases).

Don't know what's the best thing to do, maybe delete this file? Bear in mind we keep a [changelog in the RStudio _helm chart_](https://github.com/ministryofjustice/analytics-platform-helm-charts/blob/master/charts/rstudio/CHANGELOG.md). Don't know, but I thought it was worth point it out, even if we decide to keep this file.

Reply:
Yeah good point, maybe lets just keep a list of changes when we create a new release?